### PR TITLE
feat: add terraform 0.15 support and update github flows

### PR DESCRIPTION
**Pull request by syphernl**
## what
* Replaced removed `list` function with `tolist` to make it work with Terraform 0.15.
* Updated Github Workflows

## why
* The `list` function was deprecated since Terraform v0.12 and has been removed in Terraform v0.15, resulting in an `Error: Error in function call`.

### CODE REVIEW

**Review by Gowiem — APPROVED** [review]

**Comment by Gowiem** [comment]
@syphernl we need an approval from @cloudposse/admins before getting this merged. I just pinged the group. Shouldn't be long. Thanks for the contribution!

**Review by aknysh — APPROVED** [review]

**Pull request merged**
